src/Lykke.Job.TransactionHandler.Services/Offchain/OffchainRequestService.cs

```diff
@@ -38,7 +38,7 @@ public OffchainRequestService(IOffchainRequestRepository offchainRequestReposito
             var pushSettings = await _clientSettingsRepository.GetSettings<PushNotificationsSettings>(clientId);
             if (pushSettings.Enabled)
             {
-                var clientAcc = await _clientAccountsRepository.GetByIdAsync(clientId);
+                var clientAcc = await _clientAccountService.GetByIdAsync(clientId) as Lykke.Service.ClientAccount.Client.AutorestClient.Models.ClientResponseModel;
```

Comment:
Use explicit cast, or you will get NullReferenceException, when using clientAcc, instead of InvalidCastException right here. Do not defer error.